examples/tests/adc/main.c

```diff
@@ -21,7 +24,7 @@ static void test_single_samples(uint8_t channel) {
   } else {
     // 12 bit, reference = VCC/2, gain = 0.5
     // millivolts = ((sample * 2) / (2^12 - 1)) * (3.3 V / 2) * 1000
-    int millivolts = (sample * 3300) / 4095;
+    int millivolts = ((sample >> (16 - resolution)) * reference_voltage) / ((1 << resolution) - 1);
```

Comment:
The equation should just be `sample * reference_voltage / ((1 << resolution) -1)` right? https://learn.sparkfun.com/tutorials/analog-to-digital-conversion/relating-adc-value-to-voltage

I'm not sure where the `>> (16 - resolution)` comes from.

Reply:
Okay, so given that all samples essentially are 16-bit resolution (given that they are left justified) removing the `resolution` part of this altogether seems like the solution. Sorry for the confusion on my part!

Reply:
I did not use the resolution initially, I'll revert it.

Reply:
I reverted it.